resources/js/mixins.js

```diff
@@ -16,4 +16,20 @@ Vue.mixin({
             }
         },
     },
+
+    computed: {
+        fixedProductTaxes() {
+            let taxes = {}
+            this.$root.cart.items.forEach((item) => {
+                item.prices.fixed_product_taxes.forEach((tax) => {
+                    if (tax.label in taxes) {
+                        taxes[tax.label] += tax.amount.value
+                    } else {
+                        taxes[tax.label] = tax.amount.value
+                    }
+                })
+            })
+            return taxes
+        },
```

Comment:
Is this something we'd want to move to the cartStorage instead?
Like we have done with virtualitems computed properties
https://github.com/rapidez/core/blob/master/resources/js/stores/useCart.js#L99